tests/smoke/test_ucc_build.py

```diff
@@ -224,7 +224,7 @@ def test_ucc_generate_with_everything():
             ("static", "appIconAlt_2x.png"),
         ]
         for f in files_to_exist:
-            expected_file_path = path.join(expected_folder, *f)
+            expected_file_path = path.join(actual_folder, *f)
```

Comment:
nice catch, let's rename `expected_file_path` to `actual_file_path` here as well